x-pack/test/endpoint_api_integration_no_ingest/ftr_provider_context.d.ts

```diff
@@ -0,0 +1,11 @@
+/*
```

Comment:
Just curious why you had to create this typescript definition file.

Reply:
Yeah it wasn't necessary, it seemed like most test directories had their own but I think it's only useful if we define our own services and page_objects.